src/components/oidc-callback/call-back-controller.ts

```diff
@@ -47,6 +47,9 @@ export function oidcAuthCallbackGet(
 ): ExpressRouteFunc {
   return async function (req: Request, res: Response) {
     const queryParams: CallbackParamsType = req.oidc.callbackParams(req);
+    if(queryParams?.error === OIDC_ERRORS.ACCESS_DENIED) {
```

Comment:
Missing space (that prettier doesn't catch?)
```suggestion
    if (queryParams?.error === OIDC_ERRORS.ACCESS_DENIED) {
```